external-crates/move/crates/move-compiler/tests/move_check/parser/attribute_no_closing_bracket.exp

```diff
@@ -6,3 +12,12 @@ error[E01002]: unexpected token
 4 │     fun foo() {}
   │     ^ Expected ']'
 
+error[E01002]: unexpected token
+  ┌─ tests/move_check/parser/attribute_no_closing_bracket.move:4:5
+  │
+4 │     fun foo() {}
+  │     ^^^
+  │     │
+  │     Unexpected 'fun'
+  │     Expected ','
```

Comment:
Shouldn't `fun` be in the follow set for parsing attributes, avoiding this error?